build.gradle

```diff
@@ -208,8 +208,8 @@ task generateVersionFile() {
   inputs.property('gradleEnterpriseMavenExtensionVersion', gradleEnterpriseMavenExtensionVersion)
   inputs.property('commonCustomUserDataMavenExtensionVersion', commonCustomUserDataMavenExtensionVersion)
 
-  outputs.dir(versionsDir)
-      .withPropertyName('versionsDir')
+    outputs.dir(versionsDir)
```

Comment:
Why change the indentation of `ouputs.dir` here?

Reply:
@wolfs reverted